pom.xml

```diff
@@ -63,9 +63,9 @@
     </properties>
 
     <scm>
-        <connection>scm:git:git://github.com/hazelcast/hazelcast-wm.git</connection>
-        <developerConnection>scm:git:git@github.com:hazelcast/hazelcast-wm.git</developerConnection>
-        <url>https://github.com/hazelcast/hazelcast/</url>
+        <connection>scm:git:git://github.com/jd185368/cxp-hazelcast-wm.git</connection>
```

Comment:
I think we should consider moving this repository to ncr-swt-banking org once the GitHub migration is complete.

Reply:
yes.. i was trying to create it in ncr-swt-banking itself but we are not allowed to fork repos in it.. so temporarily i've created in my space.. will move it to ncr-swt-banking post github migration

Reply:
Okay. I will add a story in the backlog to move this to the Github Org once the migration is completed.